source/authentication/custom-jwt.txt

```diff
@@ -101,106 +132,79 @@ Choose your preferred method below.
-
-Audience
-~~~~~~~~
-
-The :guilabel:`Audience` of a JWT specifies its intended recipient. JWTs
-describe their audience in the ``aud`` claim. By default, App Services
-expects ``aud`` to contain the App ID of the App for which the provider
-is configured.
-
-If the external authentication system JWT specifies a different ``aud``
-value, then you can configure the provider to use that value instead.
-
-You can input a single audience or multiple audiences as a
-comma-separated list. If you add multiple audiences, you must specify
-how to handle them, either:
-
-- **All of these audiences**: the JWT must include every audience in the
-  list.
-
-- **Any one of these audiences**: the JWT only needs to include one
-  audience from the list.
 
-.. tabs-realm-admin-interfaces::
-
-   .. tab::
-      :tabid: ui
-
-      To override the default audience, specify a new value in the
-      :guilabel:`Audience` input:
-
-      .. figure:: /images/custom-jwt-audience.png
-         :alt: The Custom JWT audience configuration input
-
-      To require or allow for multiple audiences, 
-      specify multiple new values and toggle the :guilabel:`Requires` option:
+.. _custom-jwt-authentication-configuration-verification-method:
 
-      .. figure:: /images/custom-jwt-audience-multiple.png
-         :alt: The Custom JWT audience configuration input with multiple audiences
+Verification Method
+~~~~~~~~~~~~~~~~~~~
 
-   .. tab::
-      :tabid: cli
+The :guilabel:`Verification Method` field determines how App Services will 
+validate the JWT returned from the JWT provider. You can configure App Services 
+to validate the JWT by using the signing key(s) you provide, or to validate by 
+using a JSON Web Key Set (JWKS) URI issued by the third party provider.
 
-      To override the default audience, set the value of
-      ``config.audience``:
+.. _custom-jwt-authentication-configuration-manually-signing-keys:
 
-      .. code-block:: json
+Manually Specify Signing Keys
+`````````````````````````````
+You can configure your app to use one or more signing keys to validate the JWT. 
+There are two settings you need to provide:
 
-         "config": {
-           "audience": [
-             "<JWT Audience>",
-           ],
-           "requireAnyAudience": <boolean>,
-         }
-         
-      .. list-table::
-        :header-rows: 1
-        :widths: 15 30
+.. list-table::
+   :header-rows: 1
+   :widths: 15 30
 
-        * - Field
-          - Description
+   * - Field
+     - Description
 
-        * - ``audience``
+   * - | :guilabel:`Signing Algorithm`
+       | *config.signingAlgorithm*
 
-          - Array of Strings. The audience or audiences expected
-            to be found in a client JWT. 
+     - The cryptographic method that the external system uses to sign
+       the JWT. Custom authentication supports JWTs signed using any of
+       the following algorithms:
 
-        * - ``requireAnyAudience``
+       - ``HS256``
+       - ``RS256``
```

Comment:
Maybe worth adding text that says RS256 is required for JWKS?

Reply:
Not here, since this section is only about using the manual setting. Added more info in the JWK section, though.